actionsystemtest/lib/action_system_test.rb

```diff
@@ -0,0 +1,67 @@
+# System tests are similar to Integration tests in that they incorporate multiple
+# controllers and actions, but can be used to simulate a real user experience.
+# System tests are also known as Acceptance tests.
```

Comment:
Wouldn't confuse our vocabulary with talking about acceptance tests anywhere. There are many other names for system tests (including, confusingly enough, integration tests). Let's put forward a strong, consistent vocabulary.